playbook-website/app/javascript/components/Website/src/components/Scaffold.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
```

Comment:
How is this scaffold used? I'm not sure I'm following your naming. Seems like this is only used on the components pages/views?

Reply:
It's not currently being used. I'll remove. 
I have it referenced for when we do the categories view in a follow-up story. But for now, since we aren't using, I'll go ahead and remove this file.